hollow/src/main/java/com/netflix/hollow/api/codegen/HollowConsumerJavaFileGenerator.java

```diff
@@ -30,14 +40,15 @@
     protected String className;
     protected boolean useCollectionsImport=false;
 
-    public HollowConsumerJavaFileGenerator(String packageName, String subPackageName, CodeGeneratorConfig config) {
+    public HollowConsumerJavaFileGenerator(String packageName, String subPackageName,
```

Comment:
I assume this is auto formatting.   If so, it would be best only to auto format on lines that were changed.  This way it will minimize diff change noises ;)

Reply:
Oops, will revert, thanks!